lib/profileMeters.js

```diff
@@ -0,0 +1,86 @@
+/*!
+ * Copyright (c) 2020-2021 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const assert = require('assert-plus');
+const bedrock = require('bedrock');
+const database = require('bedrock-mongodb');
+
+const {util: {BedrockError}} = bedrock;
+
+bedrock.events.on('bedrock-mongodb.ready', async () => {
+  await database.openCollections(['profile-meters']);
```

Comment:
Naming convention is to use singular:

```suggestion
  await database.openCollections(['profile-meter']);
```